docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Intermediary container for building artifacts
+FROM alpine:3.14 as v86-builder
+WORKDIR /v86
+
+# Install runtime dependencies for building artifacts
+RUN apk add --update curl clang make openjdk8 npm python3
+
+# Install Rustup from Official Release using Install Shell Script
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+# Update `PATH` variable to point to Rust Binaries (Does not work automatically) and install required Crates for the target platform
+RUN PATH="${HOME}/.cargo/bin:${PATH}" rustup target add wasm32-unknown-unknown
```

Comment:
See the style of the [other Dockerfile](https://github.com/copy/v86/blob/master/tools/docker/test-image/Dockerfile). Adjacent `RUN` lines should be combined to avoid unnecessary intermediate images. Comments are only necessary for non-obvious lines (I'd say no comments are necessary for this Dockerfile).